runtime/service.go

```diff
@@ -500,10 +501,35 @@ func (s *service) createVM(requestCtx context.Context, request *proto.CreateVMRe
 	rpcClient := ttrpc.NewClient(conn, ttrpc.WithOnClose(func() { _ = conn.Close() }))
 	s.agentClient = taskAPI.NewTaskClient(rpcClient)
 	s.eventBridgeClient = eventbridge.NewGetterClient(rpcClient)
+	s.driveMountClient = drivemount.NewDriveMounterClient(rpcClient)
 	s.exitAfterAllTasksDeleted = request.ExitAfterAllTasksDeleted
 
+	err = s.mountDrives(requestCtx, request.DriveMounts)
+	if err != nil {
+		return err
+	}
+
 	s.logger.Info("successfully started the VM")
+	return nil
+}
 
+func (s *service) mountDrives(requestCtx context.Context, driveMounts []*proto.FirecrackerDriveMount) error {
+	for _, driveMount := range driveMounts {
+		driveID, err := s.stubDriveHandler.PatchStubDrive(requestCtx, s.machine, driveMount.HostPath)
```

Comment:
`PatchStubDrive` can return a `string` now for ease of use.

Reply:
Done